[2025-02-05T06:38:28.038+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:38:28.106+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task_group_example.data_processing.transform_tasks.transform_a scheduled__2024-02-12T00:00:00+00:00 [queued]>
[2025-02-05T06:38:28.110+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task_group_example.data_processing.transform_tasks.transform_a scheduled__2024-02-12T00:00:00+00:00 [queued]>
[2025-02-05T06:38:28.110+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-02-05T06:38:28.114+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): data_processing.transform_tasks.transform_a> on 2024-02-12 00:00:00+00:00
[2025-02-05T06:38:28.117+0000] {standard_task_runner.py:57} INFO - Started process 1465 to run task
[2025-02-05T06:38:28.118+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'task_group_example', 'data_processing.transform_tasks.transform_a', 'scheduled__2024-02-12T00:00:00+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/task_group_dag.py', '--cfg-path', '/tmp/tmpzo6_m00f']
[2025-02-05T06:38:28.119+0000] {standard_task_runner.py:85} INFO - Job 169: Subtask data_processing.transform_tasks.transform_a
[2025-02-05T06:38:28.126+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:38:28.139+0000] {task_command.py:415} INFO - Running <TaskInstance: task_group_example.data_processing.transform_tasks.transform_a scheduled__2024-02-12T00:00:00+00:00 [running]> on host 3849bfb82152
[2025-02-05T06:38:28.143+0000] {logging_mixin.py:151} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-05T06:38:28.168+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task_group_example' AIRFLOW_CTX_TASK_ID='data_processing.transform_tasks.transform_a' AIRFLOW_CTX_EXECUTION_DATE='2024-02-12T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-12T00:00:00+00:00'
[2025-02-05T06:38:28.168+0000] {logging_mixin.py:151} INFO - 执行任务: 转换A
[2025-02-05T06:38:28.168+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-02-05T06:38:28.171+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=task_group_example, task_id=data_processing.transform_tasks.transform_a, execution_date=20240212T000000, start_date=20250205T063828, end_date=20250205T063828
[2025-02-05T06:38:28.221+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-02-05T06:38:28.232+0000] {taskinstance.py:2784} INFO - 1 downstream tasks scheduled from follow-on schedule check
